### ISSUE TITLE
python setup.py install fails with setuptools==20.7.0

### ISSUE THREAD

**Issue by ishidawataru** [SethMMorton/natsort#55]
## Minimum, Complete, Verifiable Example

python 2.7.12
setuptools 20.7.0 (Ubuntu 16.04 apt default version)
natsort 5.3.1

Either upgrading setuptools to newer version (39.1.0) or pass `version` and `name` variable
to `setup()` in `setup.py` fixes the problem.

If `natsort` doesn't support old version of `setuptools`, it would be better to note it under
https://github.com/SethMMorton/natsort#requirements

See https://stackoverflow.com/help/mcve for explanation.

## Error message, Traceback, Desired behavior, Suggestion, Request, or Question

```
$ sudo python setup.py install
running install
running bdist_egg
running egg_info
writing src/UNKNOWN.egg-info/PKG-INFO
writing top-level names to src/UNKNOWN.egg-info/top_level.txt
writing dependency_links to src/UNKNOWN.egg-info/dependency_links.txt
reading manifest file 'src/UNKNOWN.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no previously-included files matching '*.py[cod]' found anywhere in distribution
warning: no previously-included files matching '__pycache__' found anywhere in distribution
warning: no previously-included files matching '*.so' found anywhere in distribution
writing manifest file 'src/UNKNOWN.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/utils.py -> build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/unicode_numbers.py -> build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/__main__.py -> build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/ns_enum.py -> build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/__init__.py -> build/bdist.linux-x86_64/egg/natsort
creating build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/locale.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/__init__.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/fake_fastnumbers.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/py23.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/fastnumbers.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/compat/pathlib.py -> build/bdist.linux-x86_64/egg/natsort/compat
copying build/lib.linux-x86_64-2.7/natsort/_version.py -> build/bdist.linux-x86_64/egg/natsort
copying build/lib.linux-x86_64-2.7/natsort/natsort.py -> build/bdist.linux-x86_64/egg/natsort
byte-compiling build/bdist.linux-x86_64/egg/natsort/utils.py to utils.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/unicode_numbers.py to unicode_numbers.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/__main__.py to __main__.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/ns_enum.py to ns_enum.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/locale.py to locale.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/fake_fastnumbers.py to fake_fastnumbers.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/py23.py to py23.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/fastnumbers.py to fastnumbers.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/compat/pathlib.py to pathlib.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/_version.py to _version.pyc
byte-compiling build/bdist.linux-x86_64/egg/natsort/natsort.py to natsort.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying src/UNKNOWN.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying src/UNKNOWN.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying src/UNKNOWN.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying src/UNKNOWN.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist/UNKNOWN-0.0.0-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing UNKNOWN-0.0.0-py2.7.egg
Removing /usr/local/lib/python2.7/dist-packages/UNKNOWN-0.0.0-py2.7.egg
Copying UNKNOWN-0.0.0-py2.7.egg to /usr/local/lib/python2.7/dist-packages
UNKNOWN 0.0.0 is already the active version in easy-install.pth

Installed /usr/local/lib/python2.7/dist-packages/UNKNOWN-0.0.0-py2.7.egg
Processing dependencies for UNKNOWN==0.0.0
Finished processing dependencies for UNKNOWN==0.0.0
```

**Comment by SethMMorton** [comment]
Thank you for reporting this. Do you think it would be better to just note the minimum `setuptools` that is required, or support the older versions of `setuptools`?

**Comment by ishidawataru** [comment]
Supporting older versions would be better.
I believe many python projects depend on `natsort` without specifying its version.
(At least the project I'm working on https://github.com/Azure/sonic-utilities/blob/master/setup.py )
I suspect their build is now broken due to this issue.
However, it's totally up to you :)

**Comment by SethMMorton** [comment]
Can you validate on your end that this works for you? It worked in my testing on my computer.

I am going to hold off on releasing this till I get a response regarding #57, which also is a result of this most recent change of infrastructure.

**Comment by ishidawataru** [comment]
Thanks! It works.

**Comment by SethMMorton** [comment]
Included in version 5.3.2.
